package.json

```diff
@@ -28,6 +28,9 @@
   "homepage": "https://github.com/Agoric/documentation#readme",
   "dependencies": {},
   "devDependencies": {
+    "vue": "^2.6.11",
```

Comment:
Is there a reason these are added to devDependencies?

Reply:
Actually, the only change I needed was to the `package-lock.json`, so I reverted the `package.json` change.

Thanks!